src/Microsoft.FeatureManagement/FeatureFilters/TimeWindowFilter.cs

```diff
@@ -33,7 +37,26 @@ public TimeWindowFilter(ILoggerFactory loggerFactory)
         /// <returns><see cref="TimeWindowFilterSettings"/> that can later be used in feature evaluation.</returns>
         public object BindParameters(IConfiguration filterParameters)
         {
-            return filterParameters.Get<TimeWindowFilterSettings>() ?? new TimeWindowFilterSettings();
+            DateTimeOffset? Start = filterParameters["Start"] != null ? filterParameters.GetValue<DateTimeOffset>("Start") : (DateTimeOffset?)null;
```

Comment:
I like to avoid repeated literals. I.e. "Start"

Reply:
So GetValue<DateTimeOffset?>() doesn't work?

Reply:
Ah, my bad, I didn't expect GetValue<> to be so powerful. 
It seems like GetValue<DateTimeOffset>() is enough. Thank you for the suggestion.